Dockerfile

```
@@ -0,0 +1,26 @@
+# Build ditto
+# docker build . -t ditto
+
+# Run ditto
+# docker run --rm -ti ditto --help
+
+# https://hub.docker.com/_/python
+FROM python:3.6-slim-bullseye
```

This will only work on `bullseye`, right? Does it make sense to make this the following:

```suggestion
FROM python:3.6-slim
```